package.json

```diff
@@ -162,14 +164,14 @@
 	"size-limit": [
 		{
 			"path": "dist/index.js",
-			"limit": "8 kB",
+			"limit": "10 kB",
```

Comment:
Increased the size threshold to accommodate the new components (which surpassed the 8kb)